ramls/schemas/profiles/transformations.json

```diff
@@ -20,8 +20,7 @@
     },
     "transformation": {
       "description": "Mapping expression",
-      "type": "string",
-      "pattern": "(\\d{3}[\\s\\d]{2}(\\$([a-zA-Z]|[\\d]{1,2}))?)"
+      "type": "string"
```

Comment:
Just a thought, we should have left the pattern and just added an empty to the pattern basically like UUID || "".
That way in this method only if part is needed, the else part can be removed totally :https://github.com/folio-org/mod-data-export/pull/222/files#diff-dc8e8e8fd68ceeeb43c5e09450d6a1ac227847835f12cfcbc14f07645d151479R78

Reply:
Yes, only by the reason that we cannot allow saving empty values for Item record type we should do all this validation at the service layer.

Reply:
right only the item validation can be handled in the service layer, pattern can still be done here right

Reply:
So, how are you going to apply RMB validation only for Instance and Holding record types? This pattern setting into json file is global, I mean it is applied always, no matter which any other fields passed. Correct me please if I am wrong.

Reply:
RMB will validate the pattern (a regex which will allow either UUID or an empty string) .
In the validate step all we have to do is to check if **item** transformations are empty or not. WE need to make this change now. But just mentioning that it would have made the code simpler

Reply:
> RMB will validate the pattern (a regex which will allow either UUID or an empty string) .
> In the validate step all we have to do is to check if **item** transformations are empty or not. WE need to make this change now. But just mentioning that it would have made the code simpler
We cannot do validation at RMB level because we should not allow empty values for transformation in the case of the Item record type. Therefore we should check the record type at the service level, we cannot do this at RMB level.

Reply:
Much cleaner now. Thanks for the changes